api/routers/edr.py

```diff
@@ -42,47 +49,41 @@ async def get_locations(
             points=[dstore.Point(lat=coord[1], lon=coord[0]) for coord in poly.exterior.coords]
         ),
     )
-    # TODO: It is not possible to request all of the locations, because the request is too large for gRPC.
-    #  Thus it is necessary to update the datastore.
 
     # TODO: Add flag to protobuf so that it only retrieves the latest observations from the datastore.
-    # TODO: Simulate retrieval of last obs by setting a recent timestamp
     ts_response = await getObsRequest(ts_request)
 
-    platform_standard_names = defaultdict(set)
-    for obs in sorted(ts_response.observations, key=lambda obs: obs.ts_mdata.platform):
-        platform_standard_names[obs.ts_mdata.platform].add(obs.ts_mdata.instrument)
+    platform_parameters: DefaultDict[str, Set[str]] = defaultdict(set)
+    platform_coordinates: Dict[str, Tuple[float, float]] = {}
+    all_parameters: Dict[str, Parameter] = {}
+    for obs in ts_response.observations:
+        parameter = Parameter(
+            description={"en": obs.ts_mdata.title},
+            observedProperty=ObservedProperty(
+                id=f"https://vocab.nerc.ac.uk/standard_name/{obs.ts_mdata.standard_name}",
+                label={"en": obs.ts_mdata.instrument},
+            ),
+            unit=Unit(label={"en": obs.ts_mdata.unit}),
+        )
+        platform_parameters[obs.ts_mdata.platform].add(obs.ts_mdata.instrument)
+        # HACK
+        platform_coordinates[obs.ts_mdata.platform] = (obs.obs_mdata[0].geo_point.lon, obs.obs_mdata[0].geo_point.lat)
+        # TODO: Check for overwrites?
+        all_parameters[obs.ts_mdata.instrument] = parameter
 
     features = [
         Feature(
             type="Feature",
-            id=ts.ts_mdata.platform,
-            properties={"parameter-name": platform_standard_names[ts.ts_mdata.platform]},
+            id=station_id,
+            properties={"parameter-name": sorted(platform_parameters[station_id])},
             geometry=Point(
                 type="Point",
-                coordinates=(ts.obs_mdata[0].geo_point.lon, ts.obs_mdata[0].geo_point.lat),
+                coordinates=platform_coordinates[station_id],
             ),
-        )  # HACK: Assume loc the same
-        for ts in sorted(ts_response.observations, key=lambda ts: ts.ts_mdata.platform)
+        )
+        for station_id in sorted(platform_parameters.keys())  # Sort by station_id
```

Comment:
The code looks fine, but just a general comment: It is decided yet what can go into a station_id? Could it be e.g. a WIGIOS ID (which I assume would be globally unique, no?), but also e.g. a national station ID (which then would be only be unique in combination with e.g. a standard country code)?

Reply:
And what about sensors that are moving (and then by definition not stationary at all) - should they still be identified by a station_id? (Maybe deal with that later anyway I guess)

Reply:
I was thinking about combining `naming_authority` with `platform`, where `naming_authority` is `WIGOS` (or empty) if it is a WIGOS ID?

Reply:
On moving stations, they probably should not show up in the `/locations` endpoints at all, but not clear yet how to filter them. I guess we can deal with that later.